docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -13,154 +13,220 @@ isTocNested: true
+<ul class="nav nav-tabs nav-tabs-yb">
+  <li >
+    <a href="#yugabyted" class="nav-link active" id="yugabyted-tab" data-toggle="tab" role="tab" aria-controls="yugabyted" aria-selected="true">
+      <i class="fas fa-file-alt" aria-hidden="true"></i>
+      Yugabyted
+    </a>
+  </li>
+  <li>
+    <a href="#platform" class="nav-link" id="platform-tab" data-toggle="tab" role="tab" aria-controls="platform" aria-selected="false">
+      <i class="fas fa-cloud" aria-hidden="true"></i>
+      Yugabyte Platform
+    </a>
+  </li>
+</ul>
 
-### How to Create a Multi-Zone Tablespace
+<div class="tab-content">
+  <div id="yugabyted" class="tab-pane fade show active" role="tabpanel" aria-labelledby="yugabyted-tab">
+    {{% includeMarkdown "./tablespaces-yugabyted.md" /%}}
+  </div>
+  <div id="platform" class="tab-pane fade show active" role="tabpanel" aria-labelledby="platform-tab">
+    {{% includeMarkdown "./tablespaces-platform.md" /%}}
+  </div>
+</div>
 
-The following example shows how to create a multi-zone tablespace:
+After cluster creation, verify whether the nodes have been created with the given configuration by navigating to the Tablet Servers page in the YB-Master UI
 
-```sql
-CREATE TABLESPACE us_west_tablespace 
-WITH (replica_placement='{"num_replicas": 3, "placement_blocks":
-[{"cloud":"aws","region":"us-west","zone":"us-west-1a","min_num_replicas":1},
-{"cloud":"aws","region":"us-west","zone":"us-west-1b","min_num_replicas":1},
-{"cloud":"aws","region":"us-west","zone":"us-west-1c","min_num_replicas":1}]}');
-```
+![YB Master UI - Tablet Servers Page](/images/explore/tablespaces/Geo_distributed_cluster_nodes_Master_UI.png)
 
-If you create a table and use the preceding tablespace, each tablet of the table will have three replicas, as per the `num_replicas` value. Since `placement_blocks` contains three placement blocks where `min_num_replicas` is set to 1, YB-Load balancer will ensure that `aws.us-west.us-west-1a`, `aws.us-west.us-west-1b`, and `aws.us-west.us-west-1c` will each have one replica.
+## Create a single-zone table
 
-The following example demonstrates how to create a tablespace in which the value of `min_num_replicas` does not correspond to `num_replicas`:
+By default creating any tables in the cluster created above will spread all of its data across all regions. By contrast, let us create a table and constrain all of its data within a single zone using tablespaces. The placement policy that we will use can be illustrated using the following diagram:
+
+![Single Zone Table](/images/explore/tablespaces/single_zone_table.png)
+Now let us create a tablespace outlining the above placement policy and a table associated with that tablespace:
```

Comment:
Create a tablespace outlining the preceding placement policy and a table associated with that tablespace: